ui-v2/app/templates/settings.hbs

```diff
@@ -7,14 +7,16 @@
         {{/block-slot}}
         {{#block-slot 'content'}}
             <p>
-                These settings allow you to configure your browser for the Consul Web UI. Everything is saved to localstorage, and persists through visits and browser usage.
+                These settings are specific to the Consul web UI. They are saved to local storage, and persist through browser usage and visits.
```

Comment:
major nitpick: the comma after local storage shouldn't be there since "persist through browser usage and visits" doesn't work as its own sentence.

Reply:
Good spot on the comma, realised its actually comes from the v1 UI 😂 

https://github.com/hashicorp/consul/blob/f8bcfcfa466c6cf11c92d6fa77e894f1f6935a82/ui/index.html#L759

Will sort in a sec